CHANGES.rst

```diff
@@ -1,3 +1,23 @@
+2.1.0 (2023-11-08)
```

Comment:
If we're releasing on Monday let's change this date to `2023-11-13`.

Reply:
Great catch, fixed.